modules/sdk-core/src/account-lib/mpc/tss/ecdsa/types.ts

```diff
@@ -150,7 +150,7 @@ export interface SShare {
 }
 
 export interface SignCombine {
-  gShares: GShare;
+  gShare: GShare;
```

Comment:
Technically a BREAKING CHANGE due to this being out in `latest`. Is there any risk of current end users using this and having to make the change?

Reply:
ecdsa is not released yet

Reply:
Yes, this module is actively under development and is not being used yet